README.md

```diff
@@ -81,7 +81,17 @@ On z/OS, git 2.14.4 is the minimum needed.
 ### Runtime
 To use the App Server, the following is required:
 
-* **NodeJS** - v16.x up to v18.x is officially supported by the Zowe community.
+## Node Version Setup for Zowe on Windows
+
+### Choosing the Right Node Version
+For setting up Zowe, it's recommended to install Node 16.x. Although Zowe supports up to Node 18, Node 16.x is more stable and tends to work better without errors.
+
+### Installing Node on Windows
+Avoid using `nvm` for updating Node versions on Windows as it can cause bugs. Instead, delete your current version and install the exact version from the [Node.js official website](https://nodejs.org/).
```

Comment:
It can? like what?

Reply:
It's the "Common nvm errors on Windows" area below @1000TurquoisePogs 

@Atharva-Kanherkar Let's bridge these two sections together, they don't need their own. Also I would prefer some of these to be "Notes"

Reply:
Okay sure, would you like these to be notes under the node version or under the whole readme?